Backend/tests/mock_data.json

```diff
@@ -1,14 +1,26 @@
 [
   {
-   "email" : "test@mentii.me",
-   "password" : "iameight",
-   "activationId" : "12345",
-   "active": "T"
+    "email" : "test@mentii.me",
+    "password" : "iameight",
+    "activationId" : "12345",
+    "active": "T"
   },
   {
-   "email" : "test2@mentii.me",
-   "password" : "iameight2",
-   "activationId" : "abcde",
-   "active": "F"
+    "email" : "test2@mentii.me",
+    "password" : "iameight2",
+    "activationId" : "abcde",
+    "active": "F"
+  },
+  {
```

Comment:
Not sure why the first two are not hashed. The last two are hashed because i just added them and I needed to compare a supplied password with the existing hashed version thats supposed to be in the db